fineract-provider/src/main/java/org/apache/fineract/accounting/glaccount/api/GLAccountsApiResource.java

```diff
@@ -219,4 +231,21 @@ private GLAccountData handleTemplate(final GLAccountData glAccountData) {
         }
         return returnList;
     }
+
+    @GET
+    @Path("bulkimporttemplate")
+    @Produces("application/vnd.ms-excel")
+    // @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    public Response getGlAccountsTemplate(@QueryParam("glAccountId") final Long glAccountId) {
+        return bulkImportWorkbookPopulatorService.getTemplate("glaccount",null,null,
+                null,null,null,null,null,null,null,glAccountId);
```

Comment:
create one more method to get template by passing only glaccount and glAccountId